bait-files/src/main/webapp/submission.html

```diff
@@ -43,7 +44,7 @@ <h2>What do you know about this person?</h2>
 
             <li id='phone-input'>
                 <label for="phone">Phone Number:</label><br>
-                <input type="phone" id="phone-input" name="phone-input">
+                <input type="tel" placeholder="123-456-7890" pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}" id="phone-input" name="phone-input" onkeypress="checkSpace(event)">
```

Comment:
I'd think about creating a separate helper method for this one instead of using checkSpace and calling it something like keyIsValidForPhoneNumber.  And in that method you can make sure that the only valid characters are any numeric digit or the hyphen =)